identity-service/src/routes/trackListens.js

```diff
@@ -250,20 +253,52 @@ module.exports = function (app) {
           hourlyResponseData[hourSuffix] = {
             submission: Number(await redis.get(trackingRedisKeys.submission)),
             success: Number(await redis.get(trackingRedisKeys.success)),
-            time: hourSuffix
+            time: new Date(hourSuffix)
```

Comment:
Nit - I think this is probably dateSuffix cause it can be for 7 days worth of hourly timestamps